src/azure-cli/azure/cli/command_modules/synapse/manual/_params.py

```diff
@@ -102,6 +102,9 @@ def load_arguments(self, _):
             c.argument('project_name', arg_group=repository_arg_group, help='The project name to which you are connecting.')
             c.argument('tenant_id', arg_group=repository_arg_group, help='The tenant id used to connect Azure devops')
             c.argument('last_commit_id', arg_group=repository_arg_group, help='The last commit ID.')
+            c.argument('user_assigned_identity_id', options_list=['--uami-id'], nargs='+', help='The list of User-assigned Managed Identity Id for workspace.')
+            c.argument('user_assigned_identity_in_encryption', options_list=['--uami-id-in-encrypt'], help='User assigned identity resource Id used in Workspace Encryption')
+            c.argument('use_system_assigned_identity_in_encryption', options_list=['--use-sami-in-encrypt'], help='Whether use System assigned identity in Workspace Encryption. If use uami, please set True.If not, set False')
```

Comment:
Is this parameter string type or bool type?

Reply:
string type.

Reply:
Do you want to restrict the input within `['True', 'False', 'true', 'false']`?

Reply:
No, I don't want that.